docs/releases/4.1.txt

```diff
@@ -342,6 +342,11 @@ Templates
 * :tfilter:`json_script` template filter now allows wrapping in a ``<script>``
   tag without the HTML ``id`` attribute.
 
+* The :class:`cached template loader <django.template.loaders.cached.Loader>`
+   is now enabled in development, when :setting:`DEBUG` is ``True``, and
+   :setting:`OPTIONS['loaders'] <TEMPLATES-OPTIONS>` isn't specified. You may
+   specify ``OPTIONS['loaders']`` to override this, if necessary.
```

Comment:
Extra space is unnecessary:
```suggestion
  is now enabled in development, when :setting:`DEBUG` is ``True``, and
  :setting:`OPTIONS['loaders'] <TEMPLATES-OPTIONS>` isn't specified. You may
  specify ``OPTIONS['loaders']`` to override this, if necessary.
```
![image](https://user-images.githubusercontent.com/2865885/163158592-ffcf77fe-34b1-472f-977c-8315943bcfe0.png)

Maybe this should be in the _"Backwards incompatible changes in 4.1"_ section :thinking:. I'd at least add another note there.